website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -239,6 +241,8 @@ A `default_node_pool` block supports the following:
 
 * `max_pods` - (Optional) The maximum number of pods that can run on each agent. Changing this forces a new resource to be created.
 
+* `node_labels` - (Optional) A map of Kubernetes labels which should be applied to nodes in the agent pool (e.g `{"name": "value"}`).
```

Comment:
could we make this:

```suggestion
* `node_labels` - (Optional) A map of Kubernetes labels which should be applied to nodes in the Default Node Pool.
```

(since this is a Map users can imply it's:

```
foo = {
  Hello = "World"
}
```

as such we can remove this example)